serving/docker/partition/sm_neo_utils.py

```diff
@@ -11,8 +11,8 @@ class InputConfiguration(Exception):
     """Raise when SageMaker Neo interface expectation is not met"""
 
 
-class CompilationFatalError(Exception):
-    """Raise for errors encountered during the TensorRT-LLM build process"""
+class OptimizationFatalError(Exception):
```

Comment:
Can we get this naming change approved by the Neo team ? They depend on it and interpret it in their own code base.

Reply:
I will confirm